### PR TITLE
[Feat] Support Print Class Properties

### PR DESCRIPTION
**Pull request by SonyPradana**

### CODE REVIEW

File: src/Printer.php [hunk 1]
```diff
@@ -188,13 +188,13 @@ protected function printSnapshot(&$print, $content, ...$var)
...
             ) {
-                $this->printVar($print, $var, $lenght);
+                $this->printVar($print, $var, $lenght)->new_lines();
                 continue;
             }
         }
 
         if ($has_var === true && $this->EOL_var === true) {
-            $this->printVar($print, $var, $lenght);
+            $this->printVar($print, $var, $lenght)->new_lines();
```
Comment (SonyPradana) [inline src/Printer.php]: move new line in printvar

**Review by SonyPradana — COMMENTED** [review]
posible improve class object printer

File: src/Printer.php (hunk 1 above)
Comment (anggerpradana) [reply src/Printer.php]: 💅

**Review by anggerpradana — APPROVED** [review]

File: src/Styles/ClassStyle.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+use System\Console\Style\Style;
+
+class ClassStyle extends VarPrinter
+{
+    public function render(): Style
+    {
+        $obj   = (object) $this->var;
+        $class = new ReflectionClass($obj);
+
+        $var = [];
```
Comment (SonyPradana) [inline src/Styles/ClassStyle.php]: unuse property

File: src/Styles/ClassStyle.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        foreach ($class->getDefaultProperties() as $property => $value) {
+            $visible = '';
+            if ($class->getProperty($property)->isPublic()) {
+                $value   = $class->getProperty($property)->getValue($obj);
```
Comment (SonyPradana) [inline src/Styles/ClassStyle.php]: get value private, protected by using set `setAccessible`

**Review by SonyPradana — COMMENTED** [review]
ready for merge

**Review by anggerpradana — APPROVED** [review]

**Pull request merged**